### PR TITLE
[DEVX-1154] Fix junit report

### PR DESCRIPTION
**Pull request by tianfeng92**
If there is only one testsuite, it will use an array instead of an object.

### CODE REVIEW

File: src/playwright-runner.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ async function createJob (suiteName, hasPassed, startTime, endTime, args, playwr
     }
   }
 
-  generateJunitfile(cwd, suiteName, args.param.browserName);
+  generateJunitfile(cwd, suiteName, args.param.browser);
```
Comment (alexplischke) [inline src/playwright-runner.js:60]: Hm. Confused by this change due to https://github.com/saucelabs/saucectl/blob/main/internal/playwright/config.go#L70

File: src/playwright-runner.js [hunk 2]
```diff
@@ -115,9 +115,19 @@ function generateJunitfile (cwd, suiteName, browserName) {
       encoding: 'utf-8'
     }
   };
+  console.log('===============');
+  console.log('===============');
+  console.log('===============');
+  console.log('result: ', result);
+  console.log('===============');
+  console.log('===============');
+  console.log('===============');
```
Comment (alexplischke) [inline src/playwright-runner.js:124]: Are these debug lines? 😅

Comment (tianfeng92) [reply src/playwright-runner.js]: yep :P

File: src/playwright-runner.js (hunk 1 above)
Comment (tianfeng92) [reply src/playwright-runner.js]: `browserName` doesn't work. It has been moved to `browser` in this method https://github.com/saucelabs/sauce-playwright-runner/blob/master/src/utils.js#L153

**Review by alexplischke — APPROVED** [review]

**Pull request merged**